import_export/formats/base_formats.py

```diff
@@ -3,33 +3,24 @@
 from importlib import import_module
 
 try:
-    from tablib.compat import xlrd
+    import xlrd  # NOQA
 
     XLS_IMPORT = True
 except ImportError:
-    try:
-        import xlrd  # NOQA
-
-        XLS_IMPORT = True
-    except ImportError:
-        xls_warning = "Installed `tablib` library does not include"
-        "import support for 'xls' format and xlrd module is not found."
-        warnings.warn(xls_warning, ImportWarning)
-        XLS_IMPORT = False
+    xls_warning = "Installed `tablib` library does not include"
```

Comment:
@claudep since we are not importing from tablib any more, I'd say we should remove this text, just keep the line below, maybe changing it to something like `'xls' support not available, please install the xlrd package`, what do you think?